### ISSUE TITLE
convert to git submodules

### ISSUE THREAD

**Issue by wroersma** [volatilityfoundation/community#24]
If we convert the format to git submodules we won't have to update the repo every time someone updates their plugins

**Comment by gleeda** [comment]
Yeah, I second this.  That is, IF they have a git repo...  Worst case, we can create a folder for the plugins we have so far, and then submodule the ones we know have repos..

**Comment by wroersma** [comment]
Yeah I suppose it's better than nothing. I can go ahead and make the changes if that's alright with everyone.
